app/presenters/organisations/show_presenter.rb

```diff
@@ -0,0 +1,59 @@
+module Organisations
+  class ShowPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def prefixed_title
+      prefix = needs_definite_article?(org.title) ? "the " : ""
+      (prefix + org.title)
+    end
+
+    def latest
+      stories = []
+
+      org.ordered_featured_documents.each do |story|
```

Comment:
I think this could be done more simply using `.map`. We have an example of formatting data for use with the document_list component in another place in Collections: https://github.com/alphagov/collections/blob/master/app/presenters/supergroups/supergroup.rb#L67

Would be something like:

```
      stories = org.ordered_featured_documents.each.map do |story|
        data = {
          link: {
            text: story["title"],
            path: story["href"]
          },
          metadata: {
            document_type: story["document_type"]
            public_updated_at: (Date.parse(story["public_updated_at"]) if story["public_updated_at"]),
          }
        }

        data
      end
```